X_Gstart:
  wxyz:
  - 0.003670639956034168
  - -0.9912326607383092
  - -0.11067025473869438
  - -0.07208629137387833
  xyz:
  - 0.2642002809396763
  - 0.00015012041268478923
  - 0.510819138001517
X_Insertapproach:
  wxyz:
  - -0.017089111556624174
  - -0.703957638593918
  - -0.7100358845646261
  - 0.0008049608740409104
  xyz:
  - 0.47031463894536846
  - 0.026437635577076568
  - 0.45003944187178385
X_Pregrasp:
  wxyz:
  - -0.038440440039950466
  - -0.7231642447509784
  - -0.688915629277485
  - 0.030839316148195326
  xyz:
  - 0.5749354180885888
  - 0.14111472397043606
  - 0.2494221337036544
X_Preinsert:
  wxyz:
  - 0.0003146206011174998
  - 0.70302045414351
  - 0.7110731964950496
  - -0.011706890964011035
  xyz:
  - 0.5163014851326815
  - -0.09438552374171011
  - 0.3142906664257037
